process.c

```diff
@@ -4679,56 +4679,132 @@ rb_spawn(int argc, const VALUE *argv)
 
 /*
  *  call-seq:
- *     system([env,] command... [,options], exception: false)    -> true, false or nil
+ *    system([env, ] command_line, options = {}, exception: false) -> true, false, or nil
+ *    system([env, ] exe_path, *args, options  = {}, exception: false) -> true, false, or nil
  *
- *  Executes _command..._ in a subshell.
- *  _command..._ is one of following forms.
+ *  Creates a new child process by doing one of the following
+ *  in that process:
+ *
+ *  - Passing string +command_line+ to the shell.
+ *  - Invoking the executable at +exe_path+.
  *
  *  This method has potential security vulnerabilities if called with untrusted input;
  *  see {Command Injection}[rdoc-ref:command_injection.rdoc].
  *
- *  [<code>commandline</code>]
- *    command line string which is passed to the standard shell
- *  [<code>cmdname, arg1, ...</code>]
- *    command name and one or more arguments (no shell)
- *  [<code>[cmdname, argv0], arg1, ...</code>]
- *    command name, <code>argv[0]</code> and zero or more arguments (no shell)
+ *  Returns:
  *
- *  system returns +true+ if the command gives zero exit status,
- *  +false+ for non zero exit status.
- *  Returns +nil+ if command execution fails.
- *  An error status is available in <code>$?</code>.
+ *  - +true+ if the command exits with status zero.
+ *  - +false+ if the exit status is a non-zero integer.
+ *  - +nil+ if the command fails.
```

Comment:
It is not guaranteed that successful exit is status zero, although I haven't used such platforms.
Also, "fails" feels ambiguous a little, whether the command couldn't execute or exited unsuccessfully.

Reply:
'fails' => 'could not execute'.